app/models/openscap_result.rb

```diff
@@ -0,0 +1,68 @@
+require 'openscap'
+require 'openscap/ds/arf'
+require 'openscap/xccdf/benchmark'
+
+class OpenscapResult < ApplicationRecord
+  belongs_to :container_image
+  has_one    :binary_blob,           :dependent => :destroy, :autosave => true, :as => :resource
+  has_many   :openscap_rule_results, :dependent => :destroy, :autosave => true
+
+  OpenSCAP.oscap_init
```

Comment:
@moolitayer if this is required at container analysis time, the analysis can do it at the very beginning.  Requiring this class shouldn't have unexpected side effects on the system.  If someone isn't analyzing containers, is the overhead of running this code in terms of memory usage, platform compatibility (does it work on linux/mac), wall time adversely affected by this change?